packages/contracts-bedrock/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Contributing to CONTRIBUTING.md
+
+First off, thanks for taking the time to contribute!
+
+We welcome and appreciate all kinds of contributions. Refer to the Table of Contents to discover various ways you can assist, as well as the procedures we follow for each type. Before you contribute, kindly review the appropriate section; this will streamline the process for both maintainers and contributors. We're excited to see your contributions! 🎉
```

Comment:
Might be worth mentioning that large contributions should consider opening an issue first so we can make sure they don't do a lot of work for a contribution we ultimately would reject